src/Calculator.java

```diff
@@ -7,7 +7,7 @@
  */
 
 public class Calculator {
-  public Calculator (int cacheAccessTime, int mmAccessTime, int hitCount, int missCount, boolean isLoadThrough, int blockSize){
+  public Calculator (float cacheAccessTime, float mmAccessTime, int hitCount, int missCount, boolean isLoadThrough, int blockSize){
```

Comment:
why did you make changes to calculator? this will conflict with Debbie's changes in calculator kasi you basically made the feature she's in charge of mkaing

Reply:
ahh ok i will undo the changes